nilearn/decoding/fista.py

```diff
@@ -231,7 +236,7 @@ def mfista(f1_grad, f2_prox, total_energy, lipschitz_constant, w_size,
             #
             # For this reason, we rely more on the linesearch-like
             # strategy to set the dgap_tol
-            dgap_tol = abs(energy_delta) / (i + 1.)
+            dgap_tol = abs(energy_delta)
```

Comment:
ok, it seems `dgap_tol = abs(energy_delta) / (i + 1.)` was itself a heuristic, and so no 'harm' is done. Ignore previous comment.